evap/staff/forms.py

```diff
@@ -459,6 +462,12 @@ def clean_weight(self):
             self.add_error("weight", _("At least one evaluation of the course must have a weight greater than 0."))
         return weight
 
+    def clean_main_language(self):
+        main_language = self.cleaned_data.get("main_language")
+        if self.requires_decided_main_language and main_language == "x":
```

Comment:
Do we want to put the string-constant `"x"` into some constant variable (so every occurrence is easy to find and we can change it easily if we want)? I think it would be a good idea.